ui/apps/platform/src/Components/CVEStackedPill/CVEStackedPill.js

```diff
@@ -52,7 +52,7 @@ const CVEStackedPill = ({
     return (
         <div className="flex items-center w-full">
             {useScan && !hasScan && <span>{entityName} not scanned</span>}
-            {!hasCounts && <span>No CVEs</span>}
+            {(!useScan || hasScan) && !hasCounts && <span>No CVEs</span>}
```

Comment:
Why is are the extra conditions necessary?

Expanded to what those flag vars are set by, the logic is
```suggestion
            {( !( !!scanTime) || scanTime !== '-') && !hasCounts && <span>No CVEs</span>}
            
            // which reduces to
           {( !scanTime || scanTime !== '-') && !hasCounts && <span>No CVEs</span>}
           
           // where scanTime is either `false`, or anything but '-'
```

I'm just trying to understand what will be returned for `scanTime`, if you don't have the WatchImages permission, and does it only affect Watch Images?

Reply:
Tbh this is a completely unrelated change. which fixes

<img width="224" alt="Screenshot 2023-10-03 at 4 27 31 PM" src="https://github.com/stackrox/stackrox/assets/2565181/82fd9601-21e3-4ca3-b0f6-186fb102c058">

Reply:
I'll take this out of this PR and into a different PR